config/routes.rb

```diff
@@ -0,0 +1,23 @@
+Rails.application.routes.draw do
+  devise_for :users
+
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  root 'foods#index'
+
+  resources :users 
+  resources :foods
+  resources :inventories do 
+    resources :inventory_foods
+  end
+  resources :recipes do
+    resources :recipe_foods
```

Comment:
[REQUIRED] Kindly include exceptions to these resources because you are generating unnecessary routes like show. This makes your code more professional and saves resources